shard.yml

```diff
@@ -61,6 +61,9 @@ dependencies:
   pulsar:
     github: luckyframework/pulsar
     version: ~> 0.2.2
+  teeplate:
+    github: luckyframework/teeplate
```

Comment:
😆  oh well.. this should still get rid of the circular dependency

Reply:
It's only ever required by tasks and the tasks aren't required by the main `lucky.cr` file so it won't affect compile time of the app